_editorial_policies/02_editor_instructions.md

```diff
@@ -56,10 +56,24 @@ Once an editor has handled the pre-review steps described above, the review proc
 
 Following acceptance, the procedure is:
 - Make sure the metadata is complete in the submission (including institution, ORCID, abstract). 
```

Comment:
Since this is a bit complicted, let's be super-clear on who's doing what - i.e., avoid possibly ambiguous pronouns etc:
* AUTHORS to confirm metadata is complete ...

Reply:
Clarified.